addons/godot-xr-tools/user_settings/user_settings_ui.tscn

```diff
@@ -3,127 +3,130 @@
 [ext_resource type="Script" path="res://addons/godot-xr-tools/user_settings/user_settings_ui.gd" id="1"]
 
 [node name="UserSettingsUI" type="TabContainer"]
+offset_right = 214.0
+offset_bottom = 126.0
 size_flags_horizontal = 3
 size_flags_vertical = 3
 theme_override_font_sizes/font_size = 12
 script = ExtResource("1")
 
 [node name="Input" type="VBoxContainer" parent="."]
-anchors_preset = 15
-anchor_right = 1.0
-anchor_bottom = 1.0
-offset_top = 25.0
-grow_horizontal = 2
-grow_vertical = 2
+layout_mode = 2
```

Comment:
these btw I think are Godot 3.5 features. @Malcolmnixon I think it's time to make 3.5 our minimum version?

Reply:
Agreed - this may have gone in to 4.0; but I'm trying to keep the master and 4.0-dev branches in sync.